### DISCUSSION TITLE
Build Neper with libscotch

### DISCUSSION THREAD

**Discussion by okigami** [neperfepx/neper#225] [Q&A] [answered]
Hi,

I've been trying to build Neper with libscotch and it keeps failing for some reason.
Have installed libscotch-dev in Ubuntu 20.04 and tried compiling Neper with -DHAVE_LIBSCOTCH=ON

```
- Could NOT find Scotch (missing: Scotch_INCLUDE_DIR) 
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
Scotch_INCLUDE_DIR

```

**Comment by okigami** [comment, marked as answer]
Just found the problem. One needs to install libscotch from source.
It is pretty simple:

- Download it from the website https://www.labri.fr/perso/pelegrin/scotch
- Copy the Makefile.inc.x86-64_pc_linux2 from the Make.inc directory into src directory
- Rename the just copied file to Makefile.inc
- Run make on src directory
- Run sudo make install

`Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)`

Not sure though if this speeds up things by mesh partitioning.

**Reply by rquey** [reply]
I think that your libscotch was initially installed (through the system package manager) in a location not included in your `PATH` (or maybe more likely, for some reasons, not found by CMake).

Anyway, libscotch will be included in Neper's source code in the next release.
